sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -1372,7 +1372,7 @@ class Std(private val additionalNativeFunctions: Map[String, Val.Builtin] = Map.
       v match{
         case Val.Str(_, value) => Base64.getEncoder().encodeToString(value.getBytes)
         case arr: Val.Arr => Base64.getEncoder().encodeToString(arr.iterator.map(_.cast[Val.Num].value.toByte).toArray)
-        case x => Error.fail("Cannot base64 encode " + x.prettyName)
+        case x => Error.fail("Cannot base64 encode " + x.force.prettyName)
       }
     },
 
```

Comment:
In nearby code below the fold in `prune`, I think we should be forcing the value `x` at the start of `rec` before we do the matching:

https://github.com/databricks/sjsonnet/blob/877f5aa9659dc51e277e6e9c62e94cfd137dd38e/sjsonnet/src/sjsonnet/Std.scala#L1478-L1491

We can remove the `.asStrictArray` because the forcing would then be implicitly done at the very start of `rec`, saving an unnecessary array copy.

We might also potentially be able to clean up the `.map(identity)` there, which I think was vestigial / unneeded (spotted while profiling related code recently).